config/initializers/rack_attack.rb

```diff
@@ -12,8 +12,8 @@ class Rack::Attack
     request.referer =~ spammer_regexp
   end
 
-  # Throttle room creation
-  throttle('new/rooms', :limit => 2, :period => 24.hours) do |req|
-    req.post? && req.path == "/rooms"
-  end
+  # # Throttle room creation
+  # throttle('new/rooms', :limit => 2, :period => 24.hours) do |req|
+  #   req.post? && req.path == "/rooms"
+  # end
```

Comment:
No need to comment out code; that's what version control is for, after all. We can just remove it.